tests/unit/lms/services/blackboard_api_test.py

```diff
@@ -57,6 +60,71 @@ def test_if_theres_no_refresh_token_or_expires_in(
             sentinel.access_token, None, None
         )
 
+    def test_list_files(self, svc, http_service):
+        http_service.get.return_value.validated_data = [
+            {
+                "created": "2021-06-02T15:39:51.000Z",
+                "creatorId": "_26_1",
+                "downloadUrl": "https://blackboard.example.com/downloadUrl1",
+                "id": "_7851_1",
+                "mimeType": "application/pdf",
+                "modified": "2021-06-02T15:39:51.000Z",
+                "name": "File_1.pdf",
+                "parentId": "_5801_1",
+                "size": 13472644,
+                "type": "File",
+            },
+            {
+                "created": "2021-05-02T15:39:51.000Z",
+                "creatorId": "_26_2",
+                "downloadUrl": "https://blackboard.example.com/downloadUrl2",
+                "id": "_7851_2",
+                "mimeType": "application/pdf",
+                "modified": "2021-05-02T15:39:51.000Z",
+                "name": "File_2.pdf",
+                "parentId": "_5801_1",
+                "size": 13472644,
+                "type": "File",
+            },
+            {
+                "created": "2021-04-02T15:39:51.000Z",
+                "creatorId": "_26_3",
+                "downloadUrl": "https://blackboard.example.com/downloadUrl3",
+                "id": "_7851_3",
+                "mimeType": "application/pdf",
+                "modified": "2021-04-02T15:39:51.000Z",
+                "name": "File_3.pdf",
+                "parentId": "_5801_1",
+                "size": 13472644,
+                "type": "File",
+            },
```

Comment:
This is what the actual files from the Blackboard API look like. Might be worth adding a local factory to generate these, to reduce test noise

Reply:
Might be worth slimming this down to the relevant fields + `"noise": "noise` or something. Otherwise it's not clear at a quick look what matters and what doesn't.

Reply:
Probably

Reply:
Done, I trimmed these down to just the fields that we care about plus one noise field